lib/telnyx/api_operations/nested_resource.rb

```diff
@@ -12,11 +12,16 @@ module APIOperations
     module NestedResource
       def nested_resource_class_methods(resource, path: nil, operations: nil)
         path ||= "#{resource}s"
+        if path.is_a? Array
+          path = path.map { |el| CGI.escape(el) }.join('/')
+        else
+          path = CGI.escape path
+        end
```

Comment:
What do you think of removing the if/else like this?

`Array(path).map { |el| CGI.escape(el) }.join('/')`

Reply:
+1

Reply:
Much cleaner!